src/components/UpcomingEventsList.tsx

```diff
@@ -1,39 +1,30 @@
 import { useEffect, useState } from "react"
 import { Box } from "@chakra-ui/react"
 
+import type { CommunityConference } from "@/lib/types"
+
 import Translation from "@/components/Translation"
 
 import { trackCustomEvent } from "@/lib/utils/matomo"
 
-import events from "../data/community-events.json"
+import CommunityConferences from "../data/community-events"
```

Comment:
(Similar to where it's declared, this isn't a component so I'd lowercase it by convention)